src/main/fc/fc_msp.c

```diff
@@ -2113,6 +2114,14 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         break;
 #endif
 
+    case MSP_SET_RTC:
```

Comment:
In case of MSP it's allowed to set time multiple times, right?

Reply:
That's right, the current API always allows setting the time via MSP by design. If an external device sends a request to adjust the time, we just allow it.